### PR TITLE
SMA 346 Multi owner ECDSA fixes upon audit

### PR DESCRIPTION
**Pull request by filmakarov**
# Summary

Fixes upon Zellic Audit.

Related Issue: SMA-346

## Change Type

- [x] Bug Fix

# Checklist

- [x] My code follows this project's style guidelines
- [x] I've reviewed my own code
- [x] I've added comments for any hard-to-understand areas
- [x] I've updated the documentation if necessary
- [x] My changes generate no new warnings
- [x] I've added tests that prove my fix is effective or my feature works
- [x] All unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published

### CODE REVIEW

File: contracts/smart-account/modules/MultiOwnedECDSAModule.sol [hunk 1]
```diff
@@ -44,8 +44,7 @@ contract MultiOwnedECDSAModule is
             revert AlreadyInitedForSmartAccount(msg.sender);
         }
         uint256 ownersToAdd = eoaOwners.length;
-        if (ownersToAdd == 0)
-            revert NoOwnersToAdd();
+        if (ownersToAdd == 0) revert NoOwnersToAdd();
```
Comment (livingrockrises) [inline contracts/smart-account/modules/MultiOwnedECDSAModule.sol:47]: what is the change here??

**Review by livingrockrises — COMMENTED** [review]
no differences

File: contracts/smart-account/modules/MultiOwnedECDSAModule.sol (hunk 1 above)
Comment (livingrockrises) [reply contracts/smart-account/modules/MultiOwnedECDSAModule.sol:47]: ok I just saw the linting commit. my bad

**Review by livingrockrises — APPROVED** [review]

**Comment by livingrockrises** [comment]
have we run all validation test cases against bundler integration?

**Comment by filmakarov** [comment]
> have we run all validation test cases against bundler integration?

just happy paths

**Review by Aboudjem — APPROVED** [review]

**Review by ankurdubey521 — APPROVED** [review]

**Pull request merged**